src/adapter/http/src/smart_protocol/messages.rs

```diff
@@ -12,6 +12,13 @@ use opendatafabric::{DatasetID, Multihash};
 use serde::{Deserialize, Serialize};
 use url::Url;
 
+use super::phases::TransferPhase;
+
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+pub const SMART_TRANSFER_PROTOCOL_CLIENT_VERSION: i32 = 1;
+pub const SMART_TRANSFER_PROTOCOL_SERVER_VERSION: i32 = 1;
```

Comment:
Why do we need to two separate versions for client and server?

Reply:
Yeah, agree, currently we don't need two separate versions